src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java

```diff
@@ -54,4 +60,32 @@ public PlayerIDDTO joinSession(@RequestBody String playerName, @RequestBody Stri
         PlayerIDDTO playerIDDTO = DTOMapper.INSTANCE.convertEntityToPlayerIDDTO(newPlayer);
         return playerIDDTO;
     }
+
+    @GetMapping("/tiles")
+    @ResponseBody
+    public List<TileDTO> getTilesInSession(@RequestBody String sessionID) {
+        List<Tile> tilesInSession = tileService.getTilesInSession(sessionID);
+        List<TileDTO> tileDTO = new ArrayList<>();
+
+        for (Tile tile : tilesInSession) {
+            tileDTO.add(DTOMapper.INSTANCE.convertEntityToTileDTO(tile));
+        }
+
+        return tileDTO;
+    }
+
+    @PutMapping("/drawTile")
```

Comment:
Why is drawTile a Put mapping? From my understanding, client asks the server for a tile that represents "drawing a tile from a stack", so doesnt GET make more sense? Also, shouldn't it return some info about the tile to client so that client knows which tile is drawn?

Reply:
To me put makes more sense because we don't send anything back to the client. So we just "put" something into the database based on what the client send to the server.

Reply:
By draw, do you mean "ziehen" or "malen"?

Reply:
ziehen

Reply:
probably bad naming, could change that 🤔 😓

Reply:
No its fine i understood it as ziehen too, naming is not a problem

Reply:
Is this resolved for you? Else just keep asking :)

Reply:
If we return nothing, that means we need a tileManager in client as well that constantly requests information about tiles in server. Do i understand it right now?

(Would be great to get a yes on this question hahaha)

Reply:
yessss!